content/W2017/W17-Office-Hours.md

```diff
@@ -5,29 +5,44 @@ Tags: office
 Author: Erin Edward
 Summary: Announcing our Office Hours for Winter 2017
 
-Monday:
+Office Location:
+MC 3013C
 
+Monday:  
 - 11 am - 1 pm
```

Comment:
The dashes aren't rendering as bullet points. I think the problem is that you used '+' for the days of the week and dashes for the times?